inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -720,17 +705,23 @@
             },
             "type": "object"
         },
-        "thesaurus_terms": {
+        "energy_ranges": {
+            "description": "Ranges of energies the record refers to",
+            "items": {
+                "minimum": 0,
+                "type": "integer"
+            },
+            "type": "array",
+            "uniqueItems": true
```

Comment:
I don't know what happens when you write `"uniqueItems": false` O_o

But you should just delete it. This is tested on the demo records (which are checked for validity), or later on Sentry on all public records.